Headers/DebugServer2/Core/SoftwareBreakpointManager.h

```diff
@@ -11,24 +11,48 @@
 #ifndef __DebugServer2_SoftwareBreakpointManager_h
 #define __DebugServer2_SoftwareBreakpointManager_h
```

Comment:
Looks to me like these are name-space based, not directory based. I think that means they're still correct. (For example, SessionThread is not in the ds2 namespace, and therefore doesn't have __DebugServer2)